Gemfile.lock

```diff
@@ -14,29 +14,9 @@ GIT
       addressable
       rails
 
-GEM
-  remote: https://rails-assets.org/
-  specs:
-    rails-assets-bootstrap-tabcollapse (0.2.7)
-    rails-assets-gemini-scrollbar (1.5.3)
-    rails-assets-intro.js (2.9.3)
-    rails-assets-jquery (3.4.1)
-    rails-assets-jquery-stickit (0.2.14)
-      rails-assets-jquery (>= 1.6)
-    rails-assets-jquery.scrollTo (2.1.2)
-      rails-assets-jquery (>= 1.8)
-    rails-assets-lodash (4.17.15)
-    rails-assets-moment (2.24.0)
-    rails-assets-perfect-scrollbar (0.8.1)
-    rails-assets-readmore (2.2.1)
-      rails-assets-jquery (>= 2.1.4)
-    rails-assets-slick-carousel (1.8.1)
-      rails-assets-jquery (>= 1.7)
-    rails-assets-tether (1.4.7)
-    rails-assets-waypoints (4.0.1)
-
```

Comment:
I think this changed because of your bundle version.
I recomend revert this and update your bundle to version `2.2.32` maybe.

Reply:
Thank you @mabras Fixed that 🙂